### PR TITLE
Describe undocumented registers 

### PR DESCRIPTION
**Pull request by IgorBagnucki**

### CODE REVIEW

File: devnotes/undocumented_registers.md [hunk 1]
```diff
@@ -0,0 +1,1062 @@
+# What this document is about?
+This document is an attempt to gather information about undocumented registers
```
Comment (macpijan) [inline devnotes/undocumented_registers.md:2]: @IgorBagnucki This document is nor perfectly clear for me at a first glance. You are saying that it gathers information about registers which:
- either are undocumented
- wither work differently than described

Which ones are undocumented?
Which ones work differently than described?
If they work differently than described - what is the description (and where) vs actual operation?

File: devnotes/undocumented_registers.md [hunk 2]
```diff
@@ -0,0 +1,1062 @@
...
+and registers that code analysis proofed to work diffrently than described.
+
+**P9N2_MCS_PORT02_MCPERF0 0x5010823**
+| Bit range | Name      | Assigned value | Description |
```
Comment (macpijan) [inline devnotes/undocumented_registers.md:6]: The description of a table. E.g. what is `assigned value`? Default value or something what we set maybe?

Comment (IgorBagnucki) [reply devnotes/undocumented_registers.md]: To clarify the document, I have renamed the column to `Value assigned in code`.

File: devnotes/undocumented_registers.md [hunk 3]
```diff
@@ -0,0 +1,1062 @@
...
+**XBUS_TX0_TXPACKS3_SLICE0_TX_MODE2_PL 0x80040C0C06010C3F** \
+**XBUS_TX0_TXPACKS3_SLICE1_TX_MODE2_PL 0x80040C0D06010C3F** \
+**XBUS_TX0_TXPACKS3_SLICE2_TX_MODE2_PL 0x80040C0E06010C3F** \
+**XBUS_TX0_TXPACKS3_SLICE3_TX_MODE2_PL 0x80040C0F06010C3F** \
+**XBUS_TX0_TXPACKS3_SLICE4_TX_MODE2_PL 0x80040C1006010C3F**
+| Bit range | Name            | Assigned value | Description |
+| --------- | --------------- | -------------- | ----------- |
+| 62        | TX_CAL_LANE_SEL | 1              |             |
+
+**XBUS_TX0_TXPACKS0_SLICE0_TX_BIT_MODE1_E_PL 0x80043C0006010C3F** \
```
[the remark this answers is not in the document]
Comment (IgorBagnucki) [reply devnotes/undocumented_registers.md]: Thanks to your comment, I have found out, that many of the registers have a description, but for a different address. To ease work with them, I have added a note pointing to the documentation.

**Pull request merged**